From: Daniel-Constantin Mierla <miconda@gmail.com>
Date: Fri, 5 Mar 2021 07:32:47 +0100
Subject: ctl: info log messages made debug

- used to troubleshoot infite loop on shut down phase
- GH #2661

(cherry picked from commit 38acfb59dde232b1a9f0ffc206fb46b320cd8e10)
---
 src/modules/ctl/io_listener.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/modules/ctl/io_listener.c b/src/modules/ctl/io_listener.c
index 93ce89c..ac8f64f 100644
--- a/src/modules/ctl/io_listener.c
+++ b/src/modules/ctl/io_listener.c
@@ -495,7 +495,7 @@ again:
 		DBG("handle_stream read: eof on %s\n", s_c->parent->name);
 		goto close_connection;
 	}
-	LM_INFO("bytes read: %d\n", bytes_read);
+	LM_DBG("bytes read: %d\n", bytes_read);
 	r->end+=bytes_read;
 	if (bytes_read && (bytes_read<r->bytes_to_go)){
 		r->bytes_to_go-=bytes_read;
@@ -515,7 +515,7 @@ again:
 			/* error while processing the packet => close the connection */
 			goto close_connection;
 		}
-		LM_INFO("bytes processed: %d\n", bytes_processed);
+		LM_DBG("bytes processed: %d\n", bytes_processed);
 		r->proc+=bytes_processed;
 		r->bytes_to_go=bytes_needed;
 		if (bytes_needed>0){
